### ISSUE TITLE
Grappler 

### ISSUE THREAD

**Issue by AzraelTWC** [Azanor/thaumcraft-suggestions#271]
Not to upset anyone, but the Grappler is bad. As a thing, its amazing and it was amazing in TC5 but TC6 its a clunky mess. It takes up a spot on my hot bar. It doesn't look good texture wise, meaning it doesn't fit the other great models in the mod. Its got loads of rendering issues and it simply is an annoyance rather than an awesome way to move around. I can see where you were going with it being more of a tech based thing, but the rope and whole look of it seems like its from a completely different mod.(immersive engineering) 

I would suggest returning this back into a foci and allow players to alter grapple distance, speed, and direction( meaning you can pull enemies too you.) 

In short the new grapple doesn't feel good to use for me and others I've spoken with. If the gauntlet is becoming too useful and it was needed to add another item into the game I can understand, but the mechanics and design of it are not quite there. I would do away with rope, and have it use its old animation. Maybe the model can have a small jar that has a wisp inside, and you use the grappler to launch the wisp out and pull yourself too it, and then storing the wisp back inside.

**Comment by Daderic** [comment]
I do agree with this. The new model is a little shabby, and it does take up more space than it used to. I think that a (somewhat) good reference for magic and stuff would be a game called Codespells. While the development has stopped, it did have some interesting ideas. The main part I would take from it would be the spacial manipulation of these 'orbs.' I would love to have the ability to control the positions of these orbs, or something similar in TC. If you could, it'd be sweet.

**Comment by MalkContent** [comment]
I'd really like it to be a focus again, too, simply because of the additional-item-in-the-hotbar thing.
Also a manipulatable tether effect could allow for interesting things, like roping down or a controllable tether distance to make it more like a worms ninja rope.
